schedule/yast/lvm_raid1/lvm+raid1_sle15_default.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name:           lvm+raid1_default
+description:    >
+  Validation of partitioning for raid1 on lvm
+  Installation of RAID1 using expert partitioner.
+# Note: Test data for different backends need to be
```

Comment:
you don't need this, this will be done later, now you can just focus on the arch/backend on your ticket duplicating or referencing same test data used for the existing yaml that will be untouched for other archs.